### PR TITLE
Some graph store cleanup (part 1)

### PR DESCRIPTION
**Pull request by ynx0**
Opening this early for visiblity.

**TODO**

- [x] combine old graph store reference doc
- [x] update scrys
- [x] update pokes
- [x] add input and output types for scries
- [x] explain agent state briefly, link to data structure overview for more info
- [x] update data structure overview (graph-update-3: new `%app` case in reference)

To be done in part 2
- [ ] hooks
	- [ ] gut out any references to hooks
	- [ ] move to separate specific page just talking about hooks
	- [ ] replace all current references to dedicated page
- [ ] refresh permalinks
- [ ] naming: graph-store vs. `%graph-store` vs. Graph Store

### CODE REVIEW

**Comment by tinnus-napbus** [comment]
yo I may have ticked or unticked a task accidentally sorry. I think I've put it back how it was

**Comment by ynx0** [comment]
@tinnus-napbus this is ready for review btw

File: content/docs/userspace/graph-store/graph-store-reference.md [hunk 1]
```diff
@@ -0,0 +1,246 @@
...
+  ==
+```
+
+Briefly,
+- `graphs` represents all graphs that graph store is aware of an is storing data about.
+- `tag-queries` is how graph store implements its tagging system, and uses it to keep track of associations between tags and the respective index.
+- `update-logs` is where graph-store stores the update-log that backs each graph, keyed by resource <!-- refrence to advanced section talking about how this is the source of truth -->
+- `archive` is where graph-store stores archived graphs. these are graphs which can no longer have updates run against them. <!-- this nuance should also be present in the advanced info -->
+
+For more information on these types, please take a look at the (data structure overview)[/docs/userspace/graph-store/data-structure-overview.md].
```
Comment (tinnus-napbus) [inline content/docs/userspace/graph-store/graph-store-reference.md]:
```suggestion
For more information on these types, please take a look at the [data structure overview](/docs/userspace/graph-store/data-structure-overview).
```

File: content/docs/userspace/graph-store/graph-store-reference.md [hunk 2]
```diff
@@ -0,0 +1,246 @@
...
+`[%remove-tag =term =uid]`
+
+- archive and unarchive a graph
+
+`[%archive-graph =resource]`
+
+`[%unarchive-graph =resource]`
+
+- manually process an update log on a resource
+  `[%run-updates =resource =update-log]`
```
Comment (tinnus-napbus) [inline content/docs/userspace/graph-store/graph-store-reference.md]:
```suggestion
Graph Store can be poked with a `graph-update`, which can:

### Add and remove a graph

- `[%add-graph =resource =graph mark=(unit mark) overwrite=?]`

- `[%remove-graph =resource]`

### Add and remove a node

- `[%add-nodes =resource nodes=(map index node)]`

- `[%remove-posts =resource indices=(set index)]`

### Add and remove signatures

- `[%add-signatures =uid =signatures]`

- `[%remove-signatures =uid =signatures]`

### Add and remove tags

- `[%add-tag =term =uid]`

- `[%remove-tag =term =uid]`

### Archive and unarchive a graph

- `[%archive-graph =resource]`

- `[%unarchive-graph =resource]`

### Manually process an update log on a resource

- `[%run-updates =resource =update-log]`
```

**Review by tinnus-napbus — CHANGES_REQUESTED** [review]
Looks good, just fixed the link and also slight structure change to pokes so it's more readable. Also - a few of the sentences don't have periods at the end and/or don't start with capital letters, particularly when they're bullet-points - these should be fixed

**Review by tinnus-napbus — APPROVED** [review]

**Pull request merged**